test/e2e_node/podresources_test.go

```diff
@@ -386,49 +401,51 @@ func podresourcesListTests(f *framework.Framework, cli kubeletpodresourcesv1.Pod
 	if sd != nil {
 		expected = []podDesc{
 			{
-				podName:  "pod-00",
-				cntName:  "cnt-00",
-				cpuCount: 1,
+				podName:    "pod-00",
+				cntName:    "cnt-00",
+				cpuRequest: "1000m",
 			},
 			{
 				podName:        "pod-01",
 				cntName:        "cnt-00",
 				resourceName:   sd.resourceName,
 				resourceAmount: 1,
-				cpuCount:       2,
+				cpuRequest:     "2000m",
 			},
 			{
-				podName: "pod-02",
-				cntName: "cnt-00",
+				podName:    "pod-02",
+				cntName:    "cnt-00",
+				cpuRequest: "0m",
 			},
 			{
 				podName:        "pod-03",
 				cntName:        "cnt-00",
 				resourceName:   sd.resourceName,
 				resourceAmount: 1,
-				cpuCount:       1,
+				cpuRequest:     "1000m",
 			},
 		}
 	} else {
 		expected = []podDesc{
 			{
-				podName:  "pod-00",
-				cntName:  "cnt-00",
-				cpuCount: 1,
+				podName:    "pod-00",
+				cntName:    "cnt-00",
+				cpuRequest: "1000m",
 			},
 			{
-				podName:  "pod-01",
-				cntName:  "cnt-00",
-				cpuCount: 2,
+				podName:    "pod-01",
+				cntName:    "cnt-00",
+				cpuRequest: "1000m",
 			},
 			{
-				podName: "pod-02",
-				cntName: "cnt-00",
+				podName:    "pod-02",
+				cntName:    "cnt-00",
+				cpuRequest: "0m",
 			},
 			{
-				podName:  "pod-03",
-				cntName:  "cnt-00",
-				cpuCount: 1,
+				podName:    "pod-03",
+				cntName:    "cnt-00",
+				cpuRequest: "1000m",
```

Comment:
just trying to understand why we are changing to use millis here? The tests dont seem to use fractional CPU request?

Reply:
In the commit here: https://github.com/kubernetes/kubernetes/pull/102989/commits/739f11b454afed3c05e905895e1422cffea69328 I have added a test case with fractional CPU request, which demonstrates the case of guaranteed pods with fractional CPU request thereby obtaining CPUs from the shared pool.